modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -16,6 +16,36 @@ New features are added to the language continuously, and occasionally, some feat
 This section lists all of the features that have been removed, deprecated, added, or extended in different Cypher versions.
 Replacement syntax for deprecated and removed features are also indicated.
 
+[[cypher-deprecations-additions-removals-5.26]]
+== Neo4j 5.26
+
+=== Deprecated features
+
+[cols="2", options="header"]
+|===
+| Feature
+| Details
+|===
+
```

Comment:
if its empty, why have it? I think we only add the sections we need under each version?

Reply:
Sure, can remove it.